packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -135,7 +140,20 @@ export function FiltersContainer() {
 
       {isPeopleFilterActive && <PeopleFilter />}
       {isEventTypeFilterActive && <EventTypeFilter />}
-
+      {(isPeopleFilterActive || isEventTypeFilterActive) && (
+        <Tooltip content={t("remove_filters")}>
+          <Button
+            color="secondary"
+            type="button"
+            onClick={() => {
+              if (isPeopleFilterActive) toggleOption({ label: "people" });
+              if (isEventTypeFilterActive) toggleOption({ label: "event_type" });
+              router.replace(pathname);
+            }}>
```

Comment:
Try to use

  const { removeAllQueryParams } = useFilterQuery();

removeAllQueryParams()

Reply:
This is much easier and we use it in TeamsFilter also

Reply:
I've added it in the latest commit, let me know if anything else is needed!